thirdeye/thirdeye-frontend/app/styles/pods/home/alerts-dashboard.scss

```diff
@@ -1,16 +1,35 @@
-.application-header {
-  display: flex;
-  justify-content: space-between;
-  align-items: baseline;
+.dashboard-container {
+  &__body {
+    background-color: #FFF;
+    margin-bottom: 24px;
+    padding: 0 15px;
+    border: 1px solid rgba(0, 0, 0, 0.1);
```

Comment:
fyi @newsummit created an `app-shade` mixin 

https://github.com/linkedin/pinot/blob/master/thirdeye/thirdeye-frontend/app/styles/shared/_functions.scss#L2

Reply:
Sure. Will use this.